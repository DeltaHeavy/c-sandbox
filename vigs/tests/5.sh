#!/bin/bash

echo -n "Test 5: ftable period.."
./ftable -p 3 tests/5.in tests/5.testout
diff tests/5.testout tests/5.out >/dev/null 2>&1
ret=$?
rm tests/5.testout
if [ $ret -ne 0 ]
then
    echo -e "\033[31mFAIL\033[0m"
    false
else 
    echo -e "\033[32mPASS\033[0m"
    true
fi
