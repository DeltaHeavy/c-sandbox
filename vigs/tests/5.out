Total chars: 5
A:         0 (  0.00%) 
B:         0 (  0.00%) 
C:         0 (  0.00%) 
D:         0 (  0.00%) 
E:         2 ( 40.00%) ****************************************
F:         0 (  0.00%) 
G:         0 (  0.00%) 
H:         0 (  0.00%) 
I:         0 (  0.00%) 
J:         0 (  0.00%) 
K:         0 (  0.00%) 
L:         0 (  0.00%) 
M:         0 (  0.00%) 
N:         1 ( 20.00%) ********************
O:         0 (  0.00%) 
P:         0 (  0.00%) 
Q:         0 (  0.00%) 
R:         0 (  0.00%) 
S:         0 (  0.00%) 
T:         2 ( 40.00%) ****************************************
U:         0 (  0.00%) 
V:         0 (  0.00%) 
W:         0 (  0.00%) 
X:         0 (  0.00%) 
Y:         0 (  0.00%) 
Z:         0 (  0.00%) 
